yt_dlp/extractor/anvato.py

```diff
@@ -296,15 +331,32 @@ def _get_anvato_videos(self, access_key, video_id):
                 'tbr': tbr or None,
             }
 
-            if media_format == 'm3u8' and tbr is not None:
+            vtt_subs, hls_subs = {}, {}
+            if media_format == 'vtt':
+                no_fmt, vtt_subs = self._extract_m3u8_formats_and_subtitles(
+                    video_url, video_id, ext='vtt', entry_protocol='m3u8_native',
+                    m3u8_id='vtt', fatal=False)
+                continue
+            elif media_format == 'm3u8' and tbr is not None:
                 a_format.update({
                     'format_id': join_nonempty('hls', tbr),
                     'ext': 'mp4',
                 })
             elif media_format == 'm3u8-variant' or ext == 'm3u8':
-                formats.extend(self._extract_m3u8_formats(
-                    video_url, video_id, 'mp4', entry_protocol='m3u8_native',
-                    m3u8_id='hls', fatal=False))
+                # For some videos the initial m3u8 URL returns JSON instead
+                try:
+                    manifest_json = self._download_json(
+                        video_url, video_id, note='Downloading manifest JSON')
+                except ExtractorError:
+                    pass
+                else:
```

Comment:
```suggestion
                manifest_json = self._download_json(
                    video_url, video_id, note='Downloading manifest JSON', errnote=False)
                if manifest_json:
```

Reply:
should this also be `fatal=False` if there's a chance it will be parsing an actual m3u8 instead of JSON?

Reply:
errnote=False implies non fatal